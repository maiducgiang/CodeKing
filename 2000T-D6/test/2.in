8
434 965 551 96 306 127 195 866 