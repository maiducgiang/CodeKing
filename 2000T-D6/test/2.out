217 193 29 3 153 -1 65 433 
2 5 19 2 2 -1 3 2 